test/tests/implicit-flow.spec.ts

```diff
@@ -11,7 +11,7 @@ test('logs in with an implicit flow', async ({ page, appUrl, authServerUrl }) =>
   await executor.login()
   await expect(executor.initializeAdapter(implicitFlow)).rejects.toMatchObject({
     error: 'unauthorized_client',
-    error_description: 'Client+is+not+allowed+to+initiate+browser+login+with+given+response_type.+Implicit+flow+is+disabled+for+the+client.'
+    error_description: 'Client is not allowed to initiate browser login with given response_type. Implicit flow is disabled for the client.'
```

Comment:
I am not 100% sure, but does this mean that format of `error_description` is also sligtly changed now as the data were previously encoded? Maybe worth to document this at least in some backwards compatibility guide for the case that people already rely on the `error_description` and they do some parsing of it in their apps?

Reply:
When I prepare the changelog for the next version, I will make sure to add this there.

Reply:
I don't think another issue is needed, I will go through each commit when compiling a changelog for the next release. I am resolving this thread :)